Codeinwp/neve
# Style Gutenberg blocks

### Summary

Make sure that the following Gutenberg Blocks looks similar on the site and in editor preview
- Separator
- Pullquote
- Verse
- Columns

### Will affect visual aspect of the product

NO

### Test instructions


- Try all blocks mentioned above one by one.
- Check them on the site, preferable in an incognito window, and in editor preview.
- It should look similar.


Closes #209
